internal/generator/vector/output/loki/loki.go

```diff
@@ -220,9 +215,26 @@ func TLSConf(o logging.OutputSpec, secret *corev1.Secret, op Options) []Element
 			tlsConf,
 		}
 	}
+	if o.Secret != nil || (o.TLS != nil && o.TLS.InsecureSkipVerify) {
+		if tlsConf := security.GenerateTLSConf(o, secret, op, false); tlsConf != nil {
+			tlsConf.NeedsEnabled = false
+			return []Element{tlsConf}
+		}
+
+	}
+
 	return []Element{}
 }
 
+func IsDefaultOutput(name string) bool {
```

Comment:
> Should logging.IsReservedOutputName() cover this use-case? 

I don't think so for the reasons listed below

> I also wonder if looping over the different name is necessary or if it could just be three constants, similar to logging.OutputNameDefault? (I don't think it is necessary to make these three constants part of the public API, though)

when we generate the output for the Default loki we name them based upon what you told us the name of your internal loki instance.  I don't see how it can be a constant since the name is out of our control

Reply:
Deploying CLF with custom app:

```
spec:
  inputs:
  - application:
      namespaces:
      - deleteme
    name: my-app

```
produced a named output of:
```
[sinks.default_my_app]
type = "loki"
inputs = ["default_my_app_dedot"]
endpoint = "https://lokistack-dev-gateway-http.openshift-logging.svc:8080/api/logs/v1/application"
out_of_order_action = "accept"
healthcheck.enabled = false
```
which means we need we should use the function you referenced to ensure we get custom, "default" outputs

Reply:
Ok, that's a configuration I was not aware of. Thanks :+1: 

In that case: only make it a function method of the package, if it is only relevant here?